Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenu.cpp

```diff
@@ -116,6 +125,18 @@ namespace AzToolsFramework
         return menuIterator->second;
     }
 
+    AZStd::optional<int> EditorMenu::GetWidgetSortKey(const AZStd::string& widgetActionIdentifier) const
+    {
+        auto widgetIterator = m_widgetToSortKeyMap.find(widgetActionIdentifier);
+
```

Comment:
uber nit: consistency, could remove line break here as in most other places it isn't there 😛